chromadb/api/types.py

```diff
@@ -51,7 +51,7 @@ def maybe_cast_one_to_many_ids(target: OneOrMany[ID]) -> IDs:
 
 # Embeddings
 Embedding = Vector
-Embeddings = List[Embedding]
+Embeddings = Union[List[Embedding], np.ndarray]
```

Comment:
I don't think we should mutate this type, since its used everywhere in the code. Let's instead broaden the type of Collection.py to include this.